compiler/damlc/daml-lf-conversion/src/DA/Daml/LFConversion.hs

```diff
@@ -1868,14 +1878,18 @@ rewriteStableQualified env q@(Qualified pkgRef modName obj) =
       Nothing -> q
       Just pkgId -> Qualified (PRImport pkgId) modName obj
 
-convertQualified :: NamedThing a => (T.Text -> t) -> Env -> a -> ConvertM (Qualified t)
-convertQualified toT env x = do
-  pkgRef <- nameToPkgRef env x
-  let modName = convertModuleName $ GHC.moduleName $ nameModule $ getName x
+convertGhcModule :: a -> Env -> GHC.Module -> ConvertM (Qualified a)
```

Comment:
```suggestion
convertModuleName :: a -> Env -> GHC.Module -> ConvertM (Qualified a)
```
Everything in this module converts from GHC to LF so adding `Ghc` doesn’t really add anything.

Reply:
ah I thought of that too but this module already has another `convertModuleName` function

```haskell
convertModuleName :: GHC.ModuleName -> LF.ModuleName
```

Also, mine converts a `Module`, which is the `ModuleName` plus the `UnitId`. But of course `convertModule` is taken and means something very different.

We could use a different naming scheme, what do you think about `qualifiedFromModule`?

#NamingThingsIsHard.

Reply:
ugh, too many module things. Maybe
1. ` convertModuleName` -- keep as is
2. `convertQualifiedModuleName`  -- what you called `convertGhcModule`?
3. `convertModule` -- keep as is

Alternatively we could also rename `convertModule` to `convertCoreModule` but not sure that helps